odoc-bench.opam

```
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+
+version: "dev"
+homepage: "http://github.com/ocaml/odoc"
+doc: "https://ocaml.github.io/odoc/"
+bug-reports: "https://github.com/ocaml/odoc/issues"
+license: "ISC"
+
+maintainer: [
+  "Daniel Bünzli <daniel.buenzli@erratique.ch>"
+  "Jon Ludlam <jon@recoil.org>"
+  "Jules Aguillon <juloo.dsi@gmail.com>"
+  "Paul-Elliot Anglès d'Auriac <paul-elliot@tarides.com>"
+]
+authors: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+  "Daniel Bünzli <daniel.buenzli@erratique.ch>"
+  "David Sheets <sheets@alum.mit.edu>"
+  "Jon Ludlam <jon@recoil.org>"
+  "Jules Aguillon <juloo.dsi@gmail.com>"
+  "Leo White <leo@lpw25.net>"
+  "Lubega Simon <lubegasimon73@gmail.com>"
+  "Paul-Elliot Anglès d'Auriac <paul-elliot@tarides.com>"
+  "Thomas Refis <trefis@janestreet.com>"
+]
+dev-repo: "git+https://github.com/ocaml/odoc.git"
+
+synopsis: "Meta package defining dependencies for the benchmark rule. Not released."
+
+depends: [
+  "astring"
+  "cmdliner" {>= "1.0.0"}
+  "cppo" {build & >= "1.1.0"}
+  "dune" {>= "3.7.0"}
+  "fpath"
+  "ocaml" {>= "4.02.0"}
+  "result"
+  "tyxml" {>= "4.3.0"}
+  "fmt"
+  "ocamlfind"
+  "bos"
+  "yojson" {>= "1.6.0"}
+  "mdx"
```

The `dune-project` file has `(using mdx 0.3)` as I believe it is the version needed to run mdx on mld files

 ```suggestion
  "mdx" {>= "0.3"}
```

---

That's not a Mdx version I believe but the version of Dune's support of it. I don't know the required version and I don't think we need to specify it.

---

You are right, it's dune support version.

Looking at mdx's changelog, mld supports starts at `2.3.0`: https://github.com/realworldocaml/mdx/blob/main/CHANGES.md?plain=1#L19

Why not putting a constraint, as we know it won't work with version `< 2.3.0` ?

```suggestion
  "mdx" {>= "2.3.0"}
```

---

Good idea.